src/main/webapp/homepage_script.js

```diff
@@ -58,24 +101,23 @@ function filterBy(category) {
     }
 }
 
-/** Function that display the help out confirmation overlay */
+/* Function that display the help out confirmation overlay */
 function helpOut(element) {
     const overlay = element.parentNode.parentNode.parentNode.getElementsByClassName("confirm-overlay");
```

Comment:
For this and the related methods, chaining `.parentNode` attributes is prone to breaking when the DOM changes. Instead, take a look at `element.closest()`, which can find the nearest ancestor element with a given selector. This should make it apparent to code readers which element you're looking for, and is less likely to break when intermediate elements come and go.

https://developer.mozilla.org/en-US/docs/Web/API/element/closest

Reply:
This is perfect for this purpose! Thanks for suggesting this - I'll change this now.